shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/viewModel/FavoritesViewModel.kt

```diff
@@ -117,7 +117,8 @@ public class FavoritesViewModel(
         var location: Position? by remember { mutableStateOf(null) }
         var now: EasternTimeInstant by remember { mutableStateOf(EasternTimeInstant.now()) }
 
-        val globalData = getGlobalData("FavoritesViewModel.getGlobalData")
+        val errorKey = "FavoritesViewModel"
+        val globalData = getGlobalData("$errorKey.getGlobalData")
```

Comment:
suggestion(non-blocking): Maybe `getGlobalData` can do the errorKey appending itself